pf4j/src/main/java/org/pf4j/ExtensionDescriptor.java

```diff
@@ -20,23 +20,12 @@
  */
 public class ExtensionDescriptor {
 
-    private int ordinal;
-    private Class<?> extensionClass;
+    public final int ordinal;
```

Comment:
I have a curiosity. Why you make the fields public and remove the accessors?
I don't say that is wrong but I am curious to  know your motivation.

Reply:
Because it's an immutable class, its variables cannot be changed after creation. It's also very simple class for POJOs which is used to combine multiple other objects and doesn't have other logic which depends on these variables.

Reply:
I know what it is an immutable class and the advantages of such construction.
My question is described in details on https://softwareengineering.stackexchange.com/questions/190955/using-public-final-rather-than-private-getters.
Thanks for your explanations.